### ISSUE TITLE
Semantics: axis-name vs interface-name

### ISSUE THREAD

**Issue by BrechtDeMan** [BrechtDeMan/WebAudioEvaluationTool#108]
I think calling one of the rating axes an 'interface', as in `<value interface-name="depth">` is a bit confusing. 
Before (too many) people rely on it, perhaps this could be renamed? 
This is only APE at the moment, or not?

**Comment by nickjillings** [comment]
Yes this is APE only. It can be renamed to axis-name. It's really only for the results XML where the metric tags are labelled with this to discern the movements caused by one axis.

**Comment by BrechtDeMan** [comment]
Makes sense. On the other hand, the different axes are actually called 'interface' as well (i.e. you add another interface node in test_create). So perhaps this naming convention works...

**Comment by nickjillings** [comment]
Perhaps the `<interface>` node itself should be `<axis>`?

**Comment by BrechtDeMan** [comment]
Makes sense to me. Somewhat reluctant to break this for legacy result files etc. - is it possible to make it flexible / backwards compatible for a while?
